tests/performance/src/main/java/org/mule/el/SimpleExpressionBenchmark.java

```diff
@@ -34,7 +35,7 @@
   @Setup
   public void setup() throws MuleException {
     muleContext = createMuleContextWithServices();
-    event = Event.builder(create(createFlow(muleContext), "")).message(of(PAYLOAD)).build();
+    event = Event.builder(create(createFlow(muleContext), fromSingleComponent("test"))).message(of(PAYLOAD)).build();
```

Comment:
you changed the used constant in this class. If it has to be `"test"`, use the constant.